src/core/Akka.TestKit/TestKitBase_Within.cs

```diff
@@ -283,7 +283,7 @@ public abstract partial class TestKitBase
 
                     if (resultTask == executionTask)
                     {
-                        ret = executionTask.Result;
+                        ret = executionTask.GetAwaiter().GetResult();
```

Comment:
Use `GetAwaiter().GetResult()` to unroll any `AggregateException` that might be thrown from inside the async Task